src/main/java/com/openlattice/chronicle/controllers/v2/ChronicleControllerV2.java

```diff
@@ -187,9 +179,18 @@ public void sendMessages(
         messageService.sendMessages( organizationId, messageDetails );
     }
 
-    @Override public void updateMessageStatus(
-            UUID organizationId, String messageId, MessageStatus messageStatus ) {
-        throw new NotImplementedException( "not implemented" );
+    @RequestMapping(
+            path = { ORGANIZATION_ID_PATH + MESSAGE_PATH + STATUS_PATH },
+            method = RequestMethod.POST )
+    @ResponseStatus( HttpStatus.NO_CONTENT )
```

Comment:
No change required. I don't see the benefit of `NO_CONTENT` over simply `OK`.

Reply:
this was in response to some errors that joe pointed out in our twillio logs a couple months ago. 

```
Twilio was unable to fetch content from: https://api.openlattice.com/bifrost/reminders/status
Error: Error reading response: Response does not contain content type
Account SID: AC51dec3a215872a12cfea7b6ce666953b
SID: SMe594701a52a54777a11c50d5ffc8a612
Request ID: 5bbae950-f591-4361-9074-ddd34770e42f
Remote Host: api.openlattice.com
Request Method: POST
Request URI: https://api.openlattice.com/bifrost/reminders/status
SSL Version: TLSv1.2
URL Fragment: true
```